scout/adapter/mongo/event.py

```diff
@@ -203,7 +203,6 @@ def add_phenotype(
             else:
                 raise ValueError("Must supply either hpo or omim term")
         except ValueError as ex:
-            ## TODO Should ve raise a more proper exception here?
```

Comment:
Yes, never leave a TODO for codefactor... 😅

Reply:
Coveralls is still down! If it wasn't for the security breach of few days ago I'd be tempted to switch to codecov.. I like it quite much!

Reply:
Perhaps it is Coveralls that is breached now? 😆

Reply:
Yes.. Their status was still green, but the queue-time graph peaks so I guess it is real.

Reply:
I don't mind switching - while I'm kind of used to coveralls, if you think codecov is better.. 👍🏻
But let's not do both.

Reply:
I think we have more urgent stuff to fix. But it might be a valid alternative if Coveralls keeps having problems

Reply:
We were fairly lucky to get the release out on time, before whatever hit codecov went here:
![Screenshot 2021-04-26 at 16 44 21](https://user-images.githubusercontent.com/758570/116103557-fe3f9700-a6af-11eb-8f60-153347d03622.png)
Maybe someone was finally fed up with having to write another test? 😉

Reply:
Hm, yes, actually sounds like a new Solarwind. Maybe they actually did both at the same time, only Coveralls reacted a little slower.

Reply:
Indeed!